app/models/folio/patron.rb

```diff
@@ -125,76 +132,68 @@ def remaining_checkouts
       borrow_limit - checkouts.length
     end
 
-    def proxy_borrower?
-      false
-    end
-
-    def sponsor?
-      false
-    end
-
-    def checkouts
-      @checkouts ||= patron_info['loans'].map { |checkout| Checkout.new(checkout) }
+    def all_fines
+      # @all_fines ||= []
+      @all_fines ||= patron_info['accounts'].map { |fine| Fine.new(fine) }
     end
 
     def fines
       all_fines
       # all_fines.reject { |fine| payment_sequence.include?(fine.sequence) }
     end
 
-    def all_fines
-      # @all_fines ||= []
-      @all_fines ||= patron_info['accounts'].map { |fine| Fine.new(fine) }
+    # TODO: figure this out for FOLIO
```

Comment:
Is there a ticket for this yet?

Reply:
https://github.com/sul-dlss/mylibrary/issues/830 - yes